korangar/src/loaders/mod.rs

```diff
@@ -24,3 +24,10 @@ pub use self::script::{ResourceMetadata, ScriptLoader};
 pub use self::server::{load_client_info, ClientInfo, ServiceId};
 pub use self::sprite::*;
 pub use self::texture::{TextureAtlasFactory, TextureLoader};
+
+pub const FALLBACK_PNG_FILE: &str = "missing.png";
+pub const FALLBACK_BMP_FILE: &str = "missing.bmp";
```

Comment:
`FALLBACK_BMP_FILE` and `FALLBACK_TGA_FILE` are still there but not used anymore. They do actually look different so I would either unify them into a `FALLBACK_TEXTURE_FILE` or properly check which one to return on fallback.

Reply:
We still use them here: 

https://github.com/vE5li/korangar/blob/e98be9b802a7c5a8b9daae05827396f4e68e93d1/korangar/src/loaders/texture/mod.rs#L100

Or do you mean that you want to differentiate when we don't find the image data here?

https://github.com/vE5li/korangar/blob/e98be9b802a7c5a8b9daae05827396f4e68e93d1/korangar/src/loaders/texture/mod.rs#L86